examples/mqtt/dcc_comms/aws_iot/sampleProp.conf

```diff
@@ -0,0 +1,26 @@
+#AWS IoT Configurations
+EdgeSystemName = "TestGatewayName"
+BrokerIP = "Broker Name or IP"
+BrokerPort = 8883
+keep_alive = 60
+ConnectDisconnectTimeout = 10
+cert_required = "CERT_REQUIRED"
+tls_version = "PROTOCOL_TLSv1_2"
+broker_root_ca_cert = "/etc/liota/mqtt/conf/ca.crt"
+broker_username = None
+broker_password = None
+edge_system_cert_file = "/etc/liota/mqtt/conf/client.crt"
```

Comment:
Shouldn't this location be /etc/liota/conf/mqtt/ ? All configurations should go together, and MQTT configurations be specific, isn't it? Also, will other DCCComms not need client.crt and client.key? Why this has to be specific to mqtt?